### PR TITLE
docs/jaeger: add jaeger production configuration

### PR DESCRIPTION
**Pull request by 1046102779**
i use elasticsearch as the jaeger backend storage,  the result as follows:

image
image

### CODE REVIEW

**Comment by 1046102779** [comment]
@orizohar please review this pr, thks.

File: daprdocs/content/en/operations/monitoring/tracing/supported-tracing-backends/jaeger.md [hunk 1]
```diff
@@ -62,8 +62,12 @@ The following steps shows you how to configure Dapr to send distributed tracing
 
 ### Setup
 
-First create the following YAML file to install Jaeger
-* jaeger-operator.yaml
+First create the following YAML file to install Jaeger, file name is `jaeger-operator.yaml`
+
+#### development/test configuration
+
+defaultly, allInOne uses memory as the backend storage. If you use the YAML configuration to deploy in production environment, it is dangerous!!!
```
Comment (AaronCrawfis) [inline daprdocs/content/en/operations/monitoring/tracing/supported-tracing-backends/jaeger.md]:
```suggestion
By default, the allInOne Jaeger image uses memory as the backend storage and it is not recommended to use this in a production environment.
```

File: daprdocs/content/en/operations/monitoring/tracing/supported-tracing-backends/jaeger.md [hunk 2]
```diff
@@ -62,8 +62,12 @@ The following steps shows you how to configure Dapr to send distributed tracing
 
 ### Setup
 
-First create the following YAML file to install Jaeger
-* jaeger-operator.yaml
+First create the following YAML file to install Jaeger, file name is `jaeger-operator.yaml`
+
+#### development/test configuration
```
Comment (AaronCrawfis) [inline daprdocs/content/en/operations/monitoring/tracing/supported-tracing-backends/jaeger.md]:
```suggestion
#### Development and test
```

File: daprdocs/content/en/operations/monitoring/tracing/supported-tracing-backends/jaeger.md [hunk 3]
```diff
@@ -80,6 +84,43 @@ spec:
         base-path: /jaeger
 ```
 
+#### production configuration
```
Comment (AaronCrawfis) [inline daprdocs/content/en/operations/monitoring/tracing/supported-tracing-backends/jaeger.md]:
```suggestion
#### Production
```

File: daprdocs/content/en/operations/monitoring/tracing/supported-tracing-backends/jaeger.md [hunk 4]
```diff
@@ -80,6 +84,43 @@ spec:
...
+
+jaeger uses elasticsearch as the backend storage, you maybe create jaeger-secret to access elasticsearch server. see [Configuring and Deploying Jaeger](https://docs.openshift.com/container-platform/4.7/jaeger/jaeger_install/rhbjaeger-deploying.html)
```
Comment (AaronCrawfis) [inline daprdocs/content/en/operations/monitoring/tracing/supported-tracing-backends/jaeger.md]: Could you clarify this? Currently unclear.

File: daprdocs/content/en/operations/monitoring/tracing/supported-tracing-backends/jaeger.md [hunk 5]
```diff
@@ -80,6 +84,43 @@ spec:
...
+
+> kubectl create secret generic jaeger-secret --from-literal=ES_PASSWORD='xxx' --from-literal=ES_USERNAME='xxx' -n ${NAMESPACE}
```
Comment (AaronCrawfis) [inline daprdocs/content/en/operations/monitoring/tracing/supported-tracing-backends/jaeger.md]: Please use multi-line command block instead of note (```)

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]

**Review by orizohar — CHANGES_REQUESTED** [review]
Please use upper case for start of sentence and names of technologies e.g. Jaeger Elasticsearch see [here](https://docs.dapr.io/contributing/contributing-docs/#style-and-tone) for guidance

**Review by msfussell — COMMENTED** [review]
Looks like you have forgotten to add the jaeger image file as part of this PR. Can you include this?

**Comment by 1046102779** [comment]
> n to a



> Looks like you have forgotten to add the jaeger image file as part of this PR. Can you include this?

ok. @msfussell @orizohar @AaronCrawfis please reveiw the PR again, thks.

**Review by orizohar — APPROVED** [review]

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**